draft-ietf-quic-http.md

```diff
@@ -552,10 +569,18 @@ an entire HTTP connection into a tunnel to a remote host. In HTTP/2, the CONNECT
 method is used to establish a tunnel over a single HTTP/2 stream to a remote
 host for similar purposes.
 
-A CONNECT request in HTTP/3 functions in the same manner as in HTTP/2. The
-request MUST be formatted as described in Section 8.3 of {{!HTTP2}}. A CONNECT
-request that does not conform to these restrictions is malformed (see
-{{malformed}}). The request stream MUST NOT be closed at the end of the request.
+A CONNECT request in HTTP/3 functions in the same manner as in HTTP/2.
+The request MUST be constructed as follows:
+
+- The ":method" pseudo-header field is set to "CONNECT"
+- The ":scheme" and ":path" pseudo-header fields are omitted
+- The ":authority" pseudo-header field contains the host and port to connect to
+  (equivalent to the authority-form of the request-target of CONNECT requests
+  (see Section 5.3 of [RFC7230]))
+- The request stream is not closed at the end of the request
```

Comment:
IIUC this PR has a change in normative text from "MUST NOT be closed" to "is not closed". The issue that this PR addresses  is marked editorial.

Reply:
Perhaps the "MUST be constructed" is intended to maintain the intent of the previous text. I don't think it achieves that because the stream lifetime is independent from the message.